### PR TITLE
zipnote: add page

### PR DESCRIPTION
**Pull request by Nikhil-Samant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

#8861

### CODE REVIEW

File: pages/common/zipnote.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# zipnote
+
+> View, add, or edit a zip archive's comments.
+> Files can also be renamed in the zip archive.
+> More information: <https://linux.die.net/man/1/zipnote>.
+
+- View the comments of zip archive:
```
Comment (gutjuri) [inline pages/common/zipnote.md:7]:
```suggestion
- View the comments on a zip archive:
```

File: pages/common/zipnote.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+`zipnote {{path/to/file.zip}}`
+
+- Extract comments of zip archive to a temporary file:
```
Comment (gutjuri) [inline pages/common/zipnote.md:11]:
```suggestion
- Extract the comments on a zip archive to a file:
```

File: pages/common/zipnote.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+`zipnote {{path/to/file.zip}} > {{path/to/file.txt}}`
+
+- Add/Update comments/filename in zip archive from a temporary file:
```
Comment (gutjuri) [inline pages/common/zipnote.md:15]:
```suggestion
- Add/Update comments/filename on/in a zip archive from a file:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]

File: pages/common/zipnote.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# zipnote
+
+> View, add, or edit a zip archive's comments.
+> Files can also be renamed in the zip archive.
+> More information: <https://linux.die.net/man/1/zipnote>.
```
Comment (kbdharun) [inline pages/common/zipnote.md:5]:
```suggestion
> More information: <https://manned.org/zipnote>.
```
We prefer manned when there isn't any official documentation available.

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr, I have a suggestion for this page.

**Comment by Nikhil-Samant** [comment]
> Welcome to tldr, I have a suggestion for this page.

Thanks @kbdharun for providing me the feedback. I have made the changes

File: pages/common/zipnote.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+- View the comments on a zip archive:
+
+`zipnote {{path/to/file.zip}}`
+
+- Extract the comments on a zip archive to a file:
+
+`zipnote {{path/to/file.zip}} > {{path/to/file.txt}}`
+
+- Add/Update comments/filename on/in a zip archive from a file:
```
Comment (navarroaxel) [inline pages/common/zipnote.md:15]: why say `filename` here?

Comment (Nikhil-Samant) [reply pages/common/zipnote.md:15]: I will correct it. It should not be adding or updating a filename.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**